### DISCUSSION TITLE
Add Mac address to node Device

### DISCUSSION THREAD

**Discussion by xtimmy86x** [dougiteixeira/proxmoxve#370] [Feature request] [answered]
It would be interesting to add the MAC address to the device associated with the main node to link it with other integrations that share the MAC address of the Proxmox server, such as the network connection.

**Comment by mZ738** [comment, marked as answer]
Done in 5.1.0 ([release notes](https://github.com/dougiteixeira/proxmoxve/releases/tag/5.1.0)): the node device carries the hardware addresses of its physical ports, read from the MAC-based interface names (`enx…`) Proxmox lists, so Home Assistant merges it with what a network integration sees of the same machine. Closing as done.
